app/uk/gov/hmrc/tariffclassificationfrontend/models/Sort.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.models
+
+import play.api.mvc.QueryStringBindable
+
+object Sort extends Enumeration {
+  type Sort = Value
+  val COMMODITY_CODE = Value("commodityCode")
+
+  implicit object bindable
```

Comment:
you have an object inside an object
it feels weird - never seen it before
shall we have a `val` here instead of an `object`?